src/tests/Microsoft.PowerFx.Interpreter.Tests/DateTimeTests.cs

```diff
@@ -0,0 +1,131 @@
+﻿// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+using System;
+using System.Collections.Generic;
+using System.Text;
+using System.Threading;
+using Microsoft.PowerFx.Types;
+using Xunit;
+
+namespace Microsoft.PowerFx.Interpreter.Tests
+{
+    public class DateTimeTests
+    {
+        private static readonly bool IsLocalUTC = TimeZoneInfo.Local.BaseUtcOffset == TimeSpan.Zero;
+
+        [Fact]
+        public async void DateTimeUTCTests()
```

Comment:
>DateTimeUTCTests [](http://example.com/codeflow?start=26&length=16)

nit: can we split this up into a few tests, just for maintainability. 
Doesn't matter exactly how, maybe at least 3. 

Generally, I like ordering the tests by complexity - having a basic one and getting increasingly complex.

Reply:
done.